src/Raven.Server/Config/Categories/EtlConfiguration.cs

```diff
@@ -34,5 +35,10 @@ public class EtlConfiguration : ConfigurationCategory
         [ConfigurationEntry("ETL.MaxFallbackTimeInSec", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public TimeSetting MaxFallbackTime { get; set; }
 
+        [Description("Maximum batch size of transformed data in megabytes that will be sent to the destination in a single batch")]
+        [DefaultValue(64)]
```

Comment:
Isn't this too big?

Reply:
We have 3 "regular" stop conditions to complete the batch and send the data:
- 30 sec duration
- 8192 docs
- 64MB of data

I believe that most cases 8192 will cause batches to complete. The default I set here - 64MB would be in my opinion mostly applicable if someone plays with attachments so I didn't limit it too much. What do you think?

BTW I'll add the info that attachments are also accounted to the configuration description

Reply:
Okay, that looks good